README.md

```diff
@@ -28,7 +28,25 @@ docker run -p 5432:5432 db
 
 ### Raw Data Fetching
 
-TODO
+For fetching raw data, the following ENV variables are needed:
+
+- `POSTGRES_USER` name of the database user
+- `POSTGRES_PASSWORD` database user's password
+- `POSTGRES_HOST` host address, IP address or DNS of the database
+- `POSTGRES_DB` database name
+- `RPC_TOKEN` RPC token used to initialize Solana client
+- `RATE_LIMIT` maximum number of RPC calls allowed per second
+
+Then, run the following commands:
+
+```sh
+docker build -t raw-data-fetching -f Dockerfile.raw-data-fetching .
+docker run -e POSTGRES_USER=<username> -e POSTGRES_PASSWORD=<password> -e POSTGRES_HOST=<host> -e POSTGRES_DB=<db_name> -e RPC_TOKEN=<rpc_token> -e RATE_LIMIT=<rate_limit> raw-data-fetching
+```
+
+The data collection works as follows. First, signatures of transactions are fetched in batches and stored in the database. Second, for each slot containing at least 1 signature, all transactions are fetched, but only those which match the previously saved signatures are kept and save in the database.
```

Comment:
Small thing... please it in this way
```
The data collection works as follows.
1) ...
2) ...
```
it looks stupid, but adds to the readibility for non developers... tech docs in B2B are almost always structured as 
1)
1.1)
1.1.1)
1.1.2)
1.2.1)
1.2.1.1)
1.2.2)
...